src/pages/FocusSinglePage.tsx

```diff
@@ -511,7 +511,7 @@ export const FocusSinglePageContent = ({
               </Sentry.ErrorBoundary>
             </div>
 
-            <div className='m-4'>
+            <div className='m-4' id='mutations1'>
```

Comment:
Why do we need this id?

Reply:
The link to the main page from the Mutation Tab should open the main page on the section with mutations, so this id was added in order to use Hashlink.